app/models/document.rb

```diff
@@ -37,21 +37,31 @@ def validate_parameters(content, mimetype)
     avm_service.validate_parameters(self, content, mimetype)
   end
 
+  def signature_validation
+    avm_service.validate self
+  end
+
   def signers
-    [
+    response = signature_validation
+
+    return [] if !response['signedObjects'] || response['signedObjects'].size != 1 || (response['unsignedObjects'] && response['unsignedObjects'].size != 0)
```

Comment:
Nemalo by to nejako radsej zlyhat ked tam napriklad je viacero podpisanych? Lebo asi nechceme hovorit ze to nie je podpisane ked je.

Reply:
dáva zmysel. Prípadne neriešiť ten počet dokumentov. Keď sa klient dostal do situácie, že ich tam je viac (čo zatiaľ cez AVM flow nevie), tak nech mu to normálne zvaliduje.

Reply:
Tak finálne som to spravil tak, že v signers sú proste všetky podpisy. Lebo tak to dáva zmysel, keď sa bavíme o celom kontakneri. Ak chce klient detajlnejšie info, zavolá GET /validate